legate/driver/config.py

```diff
@@ -206,7 +206,7 @@ def console(self) -> bool:
         return not any(opt.endswith(".py") for opt in self.user_opts)
 
     def _fixup_nocr(self, args: Namespace) -> None:
-        if self.console and not args.not_control_replicable:
+        if self.console and not args.not_control_replicable and args.nodes > 1:
```

Comment:
I think this needs to check that ranks > 1 (i.e. even single-node multi-rank runs should trigger this warning, and disable control replication).

Reply:
sure, this is a "fixup" that happens before `config.multi_node` is initialized from args (because it might alter `not_control_replicable` before that point) so we will have to duplicate the ranks computation but that seems ok.

Reply:
https://github.com/nv-legate/legate.core/pull/508/commits/20c2505d4cda24e0808b5d1836469c3692f1ae00